serverlessworkflow/modules/ROOT/pages/core/custom-functions-support.adoc

```diff
@@ -67,6 +67,8 @@ You must avoid using the `sysout` function in a production environment since it
 [[con-func-java]]
 == `java` custom function
 
+=== Function Definition
```

Comment:
@hmanwani-rh I would prefer to keep function definition, function argumentas and function return parameters as different sections

Reply:
@fjtirado If we need three different sections for Function definition, arguments and return parameters, then I'd suggest adding an introductory paragraph about these sections after  `== `java` custom function` instead of leaving it blank. A bit of context would be great.

Reply:
I see the context is clear for anyone reading this, but what do you suggest?

Reply:
I mean, I do not see the problem of grouping sections under another section without introduction

Reply:
But in any case Ill add something there

Reply:
I'd suggest something like a sentence like:

`java` custom functions contain the aspects that enable you to define your workflow service using function definition, arguments, and return parameters. 

And then the sections follow with proper flow.

Reply:
I moved one sentence from withih one section to the introduction, please let me know if that is fine